src/css/style.css

```diff
@@ -0,0 +1,141 @@
+body{
+    background-color: #f9f8f4;
+}
+
+.hero{
+    padding-left: 209px;
+    padding-right: 209px;
+    margin-top: 53px;
+    margin-bottom: 305px;
+    
```

Comment:
use a more efficient padding and margin targeting to avoid repetition ie, margin:10px 20px; etc